packages/base/src/keybindings.json

```diff
@@ -33,5 +53,175 @@
     "command": "jupytercad:selectConsoleCompleter",
     "keys": ["Enter"],
     "selector": ".jpcad-console .jp-ConsolePanel .jp-mod-completer-active"
+  },
+  {
+    "command": "jupytercad:sketch",
+    "keys": ["Accel Alt N"],
+    "selector": ".jcad-Mainview"
+  },
+  {
+    "command": "jupytercad:sketch",
+    "keys": ["Accel Alt N"],
```

Comment:
Instead of `Accel Shift` i've use `Accel Alt` as prefix to avoid conflicts as `Accel Shift` was conflicting with browser's keybindings.